book/src/access_control/intro.md

```diff
@@ -65,6 +65,8 @@ groups.
 | `idm_access_control_admins`  | write access controls                                                   |
 | `idm_account_policy_admins`  | modify account policy requirements for user authentication              |
 | `idm_group_admins`           | create and modify groups                                                |
+| `idm_mail_servers`           | read mail attributes needed to be a mail server                         |
```

Comment:
To match the `idm_<thing>_<subthing>_<action>` pattern
```suggestion
| `idm_all_mail_read`           | read mail attributes needed to be a mail server                         |
```

Reply:
Need to update the group to match, I can do this :)